test/integration/narrow/routes/reset-payment-request.test.js

```diff
@@ -0,0 +1,69 @@
+jest.mock('../../../../app/plugins/crumb')
+jest.mock('../../../../app/api')
+const { post } = require('../../../../app/api')
+jest.mock('../../../../app/auth')
+const createServer = require('../../../../app/server')
+const { schemeAdmin } = require('../../../../app/auth/permissions')
+const Boom = require('@hapi/boom')
+
+const url = '/payment-request/reset'
+const validInvoiceNumber = 'S1234567S123456V001'
+let server
+let auth
+
+describe('Reset payment request', () => {
+  beforeEach(async () => {
+    auth = { strategy: 'session-auth', credentials: { scope: [schemeAdmin] } }
+    jest.clearAllMocks()
+    server = await createServer()
+  })
+
+  afterEach(async () => {
+    await server.stop()
+  })
+
+  test('returns 403 and if no permissions in scope', async () => {
```

Comment:
`returns 403 if no permissions in scope`

Reply:
Fixed